vpr/SRC/base/ShowSetup.c

```diff
@@ -298,6 +299,7 @@ static void ShowRouterOpts(INP struct s_router_opts RouterOpts) {
 		vpr_printf_info("RouterOpts.initial_pres_fac: %f\n", RouterOpts.initial_pres_fac);
 		vpr_printf_info("RouterOpts.pres_fac_mult: %f\n", RouterOpts.pres_fac_mult);
 		vpr_printf_info("RouterOpts.max_router_iterations: %d\n", RouterOpts.max_router_iterations);
+		vpr_printf_info("RouterOpts.min_incremental_reroute_fanout: %d\n", RouterOpts.min_incremental_reroute_fanout);
```

Comment:
We should re-factor this option echoing routine; lots of duplicate code between detailed routing and global routing option print outs.  Should call a common routine to print most options and then just print the ones that are different in separate routines / lines.